### PR TITLE
Initial release of Illusion of Luanda

### PR DESCRIPTION
**Pull request by Atemo**
* **Addressed Issue(s)**: https://github.com/rathena/rathena/issues/6462

* **Server Mode**: renewal

* **Description of Pull Request**: 

Content
-------------------------------------------------------------------------------------------
Implemented the main quest, daily quests and exchange npcs related to Illusion of Luanda.

Walkthrough Conversion.

TODO
-------------------------------------------------------------------------------
- [x] Code review
- [x] English review
- [ ] Requires `NPC_EARTHQUAKE_K`
- [ ] Exchange final text
-------------------------------------------------------------------------------
If you have the english dialog please PM me on discord.

### CODE REVIEW

File: npc/re/quests/quests_16_2_illusion.txt [hunk 1]
```diff
@@ -5566,3 +5567,1427 @@ ice_d03_i,150,104,3	duplicate(Heap of Dry Soil#frozen1)	Heap of Dry Soil#frozen2
...
+		mes "[Chief]";
+		mes "We're in no position to take care of anyone right now, so take care of yourself..";
+		next;
+		cloakonnpcself();
+		cloakonnpcself( "Villager#LuE061" );
+		cloakonnpcself( "Villager#LuE062" );
+		cloakonnpcself( "Villager#LuE063" );
+		ill_luanda = 15;
+		setquest 5847;
+		mes "The chief who worried about me has disappeared. I have to tell Mr. Luanda what I've been through.";
```
Comment (aleos89) [inline npc/re/quests/quests_16_2_illusion.txt:6504]:
```suggestion
		mes "The Chief who is worried about me has disappeared. I have to tell Mr. Luanda what I've been through.";
```

File: npc/re/quests/quests_16_2_illusion.txt [hunk 2]
```diff
@@ -5566,3 +5567,1427 @@ ice_d03_i,150,104,3	duplicate(Heap of Dry Soil#frozen1)	Heap of Dry Soil#frozen2
...
+		mes "[Third Young]";
+		mes "Brothers, it looks like you hunt well this time.";
+		next;
+		npctalk "Something was a bit sinister, but I think this time it will be quite profitable, youngest.", "First Young#LuE07", bc_self;
+		mes "[First Young]";
+		mes "Something was a bit sinister, but I think it will be quite profitable this time, youngest.";
+		next;
+		npctalk "Huh? Brother, hey.. I've seen him before.", "", bc_self;
+		mes "[Third Young]";
+		mes "Huh? Brother, over there.. I saw an outsider before.";
```
Comment (aleos89) [inline npc/re/quests/quests_16_2_illusion.txt:6553]:
```suggestion
		npctalk "Huh? Brother, over there.. I've seen him before.", "", bc_self;
		mes "[Third Young]";
		mes "Huh? Brother, over there.. I've seen him before.";
```

File: npc/re/quests/quests_16_2_illusion.txt [hunk 3]
```diff
@@ -5566,3 +5567,1427 @@ ice_d03_i,150,104,3	duplicate(Heap of Dry Soil#frozen1)	Heap of Dry Soil#frozen2
...
+		next;
+		npctalk "Yeah, I guess so? That's a face I can remember too.", "First Young#LuE07", bc_self;
+		mes "[First Young]";
+		mes "Yeah, I guess so? That's a face I remember too.";
+		next;
+		npctalk "I heard that the stern-faced second warned you, but what a fearless outsider you are.", "The First Young#LuE07", bc_self;
+		mes "[First Young]";
+		mes "I heard that the stern-faced second warned you, but you're a fearless stranger.";
```
Comment (aleos89) [inline npc/re/quests/quests_16_2_illusion.txt:6561]:
```suggestion
		mes "I heard that the stern-faced second warned you, but what a fearless outsider you are.";
```

File: npc/re/quests/quests_16_2_illusion.txt [hunk 4]
```diff
@@ -5566,3 +5567,1427 @@ ice_d03_i,150,104,3	duplicate(Heap of Dry Soil#frozen1)	Heap of Dry Soil#frozen2
...
+		next;
+		npctalk "Brother.. I'm all the same..", "", bc_self;
+		mes "[Third Young]";
+		mes "Brother.. I'm all the same spirit..";
```
Comment (aleos89) [inline npc/re/quests/quests_16_2_illusion.txt:6565]:
```suggestion
		mes "Brother.. I'm all the same..";
```

File: npc/re/quests/quests_16_2_illusion.txt [hunk 5]
```diff
@@ -5566,3 +5567,1427 @@ ice_d03_i,150,104,3	duplicate(Heap of Dry Soil#frozen1)	Heap of Dry Soil#frozen2
...
+com_d02_i,39,219,6	duplicate(dummy_cloaked_npc)	First Young#LuE07	4_M_THAIONGBAK
+com_d02_i,41,220,4	duplicate(dummy_cloaked_npc)	Second Young#LuE07	4_M_THAIONGBAK
+
+
+// (re)TUESDAY
+com_d02_i,116,94,6	script	Karu#LuE08	4_M_THAIONGBAK,{
+	if (ill_luanda == 18) {
+		npctalk "Ahh! Asudem why? Why? My... how did I find you...", "", bc_self;
+		mes "[karoo]";
+		mes "Aww! Asudem why? Why? My... how did I find you...";
```
Comment (aleos89) [inline npc/re/quests/quests_16_2_illusion.txt:6599]:
```suggestion
		npctalk "Ahh! Asudem, why? Why? My... how did I find you...", "", bc_self;
		mes "[Karu]";
		mes "Aww! Asudem, why? Why? My... how did I find you...";
```

File: npc/re/quests/quests_16_2_illusion.txt [hunk 6]
```diff
@@ -5566,3 +5567,1427 @@ ice_d03_i,150,104,3	duplicate(Heap of Dry Soil#frozen1)	Heap of Dry Soil#frozen2
...
+		next;
+		npctalk "Ka.. Karu..", "Asudem#LuE08", bc_self;
+		mes "[Asudem]";
+		mes "Ka.. Karu..";
+		next;
+		npctalk "Hey hey hey Asudem... honey...", "", bc_self;
+		mes "[karoo]";
+		mes "Hey hey hey Asudem.. honey..";
```
Comment (aleos89) [inline npc/re/quests/quests_16_2_illusion.txt:6607]:
```suggestion
		npctalk "Hey, hey, hey, Asudem... honey...", "", bc_self;
		mes "[Karu]";
		mes "Hey, hey, hey, Asudem.. honey..";
```

File: npc/re/quests/quests_16_2_illusion.txt [hunk 7]
```diff
@@ -5566,3 +5567,1427 @@ ice_d03_i,150,104,3	duplicate(Heap of Dry Soil#frozen1)	Heap of Dry Soil#frozen2
...
+		mes "[Asudem]";
+		mes "Karu, don't be too sad.. You found me like this..";
+		next;
+		npctalk "I... I'm... glad...", "Asudem#LuE08", bc_self;
+		mes "[Asudem]";
+		mes "I... I'm... glad...";
+		next;
+		npctalk "This... it's not about dying... their curse... on you...", "", bc_self;
+		mes "[karoo]";
+		mes "This.. This means dying is not the problem.. Their curse.. on you..";
```
Comment (aleos89) [inline npc/re/quests/quests_16_2_illusion.txt:6619]:
```suggestion
		npctalk "This.. it's not about dying.. their curse.. on you..", "", bc_self;
		mes "[Karu]";
		mes "This.. it's not about dying.. their curse.. on you..";
```

File: npc/re/quests/quests_16_2_illusion.txt [hunk 8]
```diff
@@ -5566,3 +5567,1427 @@ ice_d03_i,150,104,3	duplicate(Heap of Dry Soil#frozen1)	Heap of Dry Soil#frozen2
...
+		next;
+		npctalk "Ka.. Karu..", "Asudem#LuE08", bc_self;
+		mes "[Asudem]";
+		mes "Ka.. Karu..";
+		next;
+		npctalk "Hey hey hey Asudem... honey...", "", bc_self;
+		mes "[karoo]";
+		mes "Hey hey hey Asudem.. honey..";
+		next;
+		npctalk "Don't be too sad, Karu.. you found me like this..", "Asudem#LuE08", bc_self;
```
Comment (aleos89) [inline npc/re/quests/quests_16_2_illusion.txt:6609]:
```suggestion
		npctalk "Karu, don't be too sad.. You found me like this..", "Asudem#LuE08", bc_self;
```

File: npc/re/quests/quests_16_2_illusion.txt [hunk 9]
```diff
@@ -5566,3 +5567,1427 @@ ice_d03_i,150,104,3	duplicate(Heap of Dry Soil#frozen1)	Heap of Dry Soil#frozen2
...
+		mes "[Asudem]";
+		mes "I... I'm... glad...";
+		next;
+		npctalk "This... it's not about dying... their curse... on you...", "", bc_self;
+		mes "[karoo]";
+		mes "This.. This means dying is not the problem.. Their curse.. on you..";
+		next;
+		npctalk "All..you are no longer..", "", bc_self;
+		mes "[karoo]";
+		mes "everything.. you no longer..";
```
Comment (aleos89) [inline npc/re/quests/quests_16_2_illusion.txt:6623]:
```suggestion
		npctalk "All.. you no longer..", "", bc_self;
		mes "[Karu]";
		mes "All.. you no longer..";
```

File: npc/re/quests/quests_16_2_illusion.txt [hunk 10]
```diff
@@ -5566,3 +5567,1427 @@ ice_d03_i,150,104,3	duplicate(Heap of Dry Soil#frozen1)	Heap of Dry Soil#frozen2
...
+		next;
+		npctalk "That's not crazy kkkk.", "", bc_self;
+		mes "[???]";
+		mes "That's not crazy kkkk.";
+		next;
+		npctalk "Once we claim the land, the legendary power will be ours too.", "Silhouette#LuE092", bc_self;
+		mes "[???]";
+		mes "Once we claim the land, the legendary power will be ours too.";
+		next;
+		npctalk "Sniff sniff... I can smell the humans. It's not time to run into them yet. Let's go.", "", bc_self;
```
Comment (aleos89) [inline npc/re/quests/quests_16_2_illusion.txt:6701]:
```suggestion
		npctalk "Sniff... I can smell the humans. It's not time to run into them yet. Let's go.", "", bc_self;
```

**Review by aleos89 — APPROVED** [review]

File: npc/re/quests/quests_16_2_illusion.txt [hunk 11]
```diff
@@ -5566,3 +5567,1427 @@ ice_d03_i,150,104,3	duplicate(Heap of Dry Soil#frozen1)	Heap of Dry Soil#frozen2
...
+
+com_d02_i,152,40,6	script	Second Young#LuE01	4_M_THAIONGBAK,{
+	if (ill_luanda == 3) {
+		npctalk "Brother? Are you ready? Are you leaving soon? Time is running out.", "", bc_self;
+		mes "[Second Young]";
+		mes "Brother? Are you ready? Are you leaving soon? Time is running out.";
+		next;
+		npctalk "Yes. As the second said, we'd better start quickly. I don't feel very good.", "First Young#LuE01", bc_self;
+		mes "[First Young]";
+		mes "Yes. As the second said, it's better to start quickly. I'm not very good at it.";
```
[the remark this answers is not in the document]
Comment (Atemo) [reply npc/re/quests/quests_16_2_illusion.txt:6113]: You mean it should be `"I don't feel very good.";`, right?

File: npc/re/quests/quests_16_2_illusion.txt [hunk 12]
```diff
@@ -5566,3 +5567,1427 @@ ice_d03_i,150,104,3	duplicate(Heap of Dry Soil#frozen1)	Heap of Dry Soil#frozen2
...
+		mes "[Luanda]";
+		mes "If so... the sister-in-law also...";
+		next;
+		select( "It's not your fault" );
+		mes "[Luanda]";
+		mes "I know. It's not because of me and my brothers.. but.. that day.. we.. .. .. it's just a pity";
+		next;
+		select( "You'd better calm down first." );
+		mes "[Luanda]";
+		mes "You've shown the adventurer something you wouldn't have seen. I'm just here to tell you what I saw...";
```
[the remark this answers is not in the document]
Comment (Atemo) [reply npc/re/quests/quests_16_2_illusion.txt]: hmm not sure, but it seems correct!

**Pull request merged**